tools/accuracy_checker/accuracy_checker/postprocessor/letterbox_postrocessing_resize.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) 2021 Intel Corporation
```

Comment:
```suggestion
Copyright (c) 2018-2021 Intel Corporation
```
we use the same license years like in openvino